lib/msf/core/exploit/remote/kerberos/ticket/storage/write_mixin.rb

```diff
@@ -0,0 +1,37 @@
+module Msf::Exploit::Remote::Kerberos::Ticket::Storage
+  # A mixin providing the ability to store new and delete existing tickets.
+  module WriteMixin
+    # (see Base#delete_tickets)
+    def delete_tickets(options = {})
```

Comment:
What are your thoughts on this method taking the array of tickets that we want to delete?

I do like the atomic query deletion that the current API offers - but specifically the functionality is required for the `klist` command which applies a filter with the `-d` flag 🤔

Reply:
Following the existing pattern that's used for the loots command; we wouldn't be able to use the `delete_tickets` api wholesale, as there's support for the user providing multiple hosts - which requires multiple lookups

```ruby
    ticket_results = []
    each_host_range_chunk(host_ranges) do |host_search|
      next if host_search && host_search.empty?

      ticket_results += get_kerberos_tickets(
        workspace: framework.db.workspace,
        hosts: { address: host_search }
      )
```